src/app/features/fee-editor/fee-editor.tsx

```diff
@@ -0,0 +1,77 @@
+import { useNavigate } from 'react-router-dom';
+
+import { Center, Stack, styled } from 'leather-styles/jsx';
+
+import { Approver, Button } from '@leather.io/ui';
+
+import { CurrentFeeItem } from './components/current-fee-item';
+import { CustomFeeItem } from './components/custom-fee-item';
+import { DefaultFeesList } from './components/default-fees-list';
+import { FeeItem } from './components/fee-item';
+import { useFeeEditorContext } from './fee-editor.context';
+
+interface FeeEditorProps {
+  children?: React.ReactNode;
+}
+function FeeEditor({ children }: FeeEditorProps) {
+  const {
+    currentEditorFee,
+    customEditorFeeRate,
+    isLoadingFees,
+    editorFees,
+    selectedEditorFee,
+    getCustomEditorFee,
+    onSetCurrentEditorFee,
+    onSetCustomEditorFeeRate,
```

Comment:
I wonder if it'd be cleaner to have a single fee setting, using a discriminating string literal?

e.g. 
```ts
// making up prop names here
onSetCurrentEditorFee({ type: 'custom', customValue: 'xxx' });
onSetCurrentEditorFee({ type: 'high' });
```